package.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -euxo pipefail
+
+git clone --branch $DECKSCHRUBBER_TAG https://github.com/fraunhoferfokus/deckschrubber.git /tmp/deckschrubber
+cd /tmp/deckschrubber
+
+go get -v
+
+# Build the .deb and put it in /mnt, which is mounted from dist_*/ in the Makefile.
+cd /mnt
+fpm -s dir -t deb \
+    -n deckschrubber \
+    -v $(echo $DECKSCHRUBBER_TAG | sed 's/[^0-9.]*//g')~ocf1 \
+    --deb-generate-changes \
+    --deb-dist $DIST_TAG \
+    --description "Deckschrubber inspects images of a Docker Registry and removes those older than a given age." \
+    --maintainer "root@ocf.berkeley.edu" \
```

Comment:
I'd add in `--url https://github.com/fraunhoferfokus/deckschrubber` (or maybe even this repo, not sure which would be more useful there), but otherwise this looks good to me.

Reply:
Thanks for setting up the pipeline! I put this repo as the URL, since it links to deckschrubber's repo in the README too.